nightingale_interface/screens/admin_screen.py

```diff
@@ -0,0 +1,89 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+
+from kivymd.uix.label import MDLabel
+from kivymd.uix.button import MDRectangleFlatButton
+from kivy.uix.button import Button
+
+from kivy.uix.screenmanager import SlideTransition, NoTransition
+
+from screens.screen_config import ScreenConfig as cfg
+
+
+class AdminScreen:
+    admin_name = "adminscreen"
```

Comment:
Is there any reason that this is a class variable instead of static? I feel like this should be a constant defined in either the config file or at the top of the screen file

Reply:
Yeah I actually made some changes to move all of those name variables to the config file since the strings are used in multiple places. The change is on the new branch though with all of the interface stuff